### DISCUSSION TITLE
Rename crate table-engine to mito

### DISCUSSION THREAD

**Discussion by evenyag** [GreptimeTeam/greptimedb#428] [General]
Now the `mito` table engine is implemented in the `table-engine` crate. I think it would be much clear if we rename this crate to `mito`, since `table-engine` sometimes is easy to confuse with the `table` crate, which defines our API for a table engine.

Using `table-engine` is much like a historical reason as we hadn't decided the name for our default table engine at the time we started to write the engine. Another benefit is `mito` only takes one word and `table-engine` takes two.

**Comment by MichaelScofield** [comment]
Aye!

**Comment by waynexia** [comment]
How about `mito-engine`? Only `mito` may look unclear that what is inside.

**Reply by evenyag** [reply]
Well, there are two options:
- `mito`
- `mito-engine`

**Reply by evenyag** [reply]
I think `mito` is enough, as `innodb`, `WiredTiger`.

**Comment by evenyag** [comment]
Closed by #539
